Source/WebKit/WebProcess/WebPage/RemoteLayerTree/RemoteScrollingCoordinator.mm

```diff
@@ -59,17 +59,21 @@
 RemoteScrollingCoordinator::RemoteScrollingCoordinator(WebPage* page)
     : AsyncScrollingCoordinator(page->corePage())
     , m_webPage(page)
+    , m_pageIdentifier(m_webPage->identifier())
```

Comment:
nit: I would call `page->identifier()` here. It doesn't matter much in this instance because `identifier()` is a trivial function but with static analysis, `page->nonTrivial()` is safe since `page` is an argument, while `m_webPage->nonTrivial()` is not and would force you to protect m_webPage.